qiskit/aqua/operators/operator_base.py

```diff
@@ -498,6 +524,24 @@ def _get_param_dict_for_index(unrolled_dict: Dict[ParameterExpression, List[Numb
         """ Gets a single non-list-nested param_dict for a given list index from a nested one. """
         return {k: v[i] for (k, v) in unrolled_dict.items()}
 
+    def _expand_shorter_operator_and_permute(self, other: 'OperatorBase',
+                                             permutation: List[int] = None) \
```

Comment:
```suggestion
                                             permutation: Optional[List[int]] = None) \
```

Reply:
This isn't resolved, could you please update this type hint? If `None` is allowed as value the type hint must include that. Either via `Union[List[int], None] = None` or `Optional[List[int]] = None`

Reply:
@Cryoris it is resolved, the suggestion is outdated and in source code, there is already Optional[List[int]] = None